src/platform/site-wide/sass/modules/_m-responsive-table.scss

```diff
@@ -14,4 +14,50 @@
       display: block;
     }
   }
+  .rwd tr {
+    border-top: 2px solid #999;
+  }
+  .rwd thead {
+    display: none;
+  }
+  .rwd td,
+  .rwd th {
+    border: none;
+    display: block;
+  }
+  .rwd td {
+    display: flex;
+    grid-gap: 2px;
+    padding: 1rem;
+    padding-left: 0px;
+  }
+  .rwd dfn {
+    display: block;
+    font-style: normal;
+    font-weight: 800;
+    margin: 0;
+    padding: 0;
+  }
+  .rwd span {
+    text-align: right;
+  }
+  .rwd tr td:nth-child(3) {
+    display: block;
+  }
+  .rwd tr td:nth-child(3) dfn {
+    display: block;
+  }
```

Comment:
What are your thoughts on `va-table--responsive` instead of `rwd`? 

I'm also wondering on the "3" in `td:nth-child(3)` - what that numbers means versus a 2 or 4

Reply:
i tried to add in va-table--responsive instead of rwd and it broke everything 😬
the `rwd` class is from jens codepen
re: the specific selector you pointed out.. it wasn't doing anything, so i removed it as well as a couple of other selectors that were specific to the structure in the codepen and didn't match the data returned from the cms.

Reply:
Oh I just meant if we could use something like `va-table--responsive ` as the class name instead of `rwd`. Off the bat, I don't know what `rwd` stands for and just thinking we could use a more descriptive name

Reply:
ah gotcha... that makes sense - done.

Reply:
Yes! Y'all should definitely change `.rwd` to use the naming convention used on VA.gov. I used `.rwd` in mine because I don't know what y'all would use (rwd stands for responsive web design, but it's totally my shorthand sketch).